src/DetailsView/components/command-bar-buttons-menu.tsx

```diff
@@ -11,25 +11,7 @@ export type CommandBarButtonsMenuProps = CommandBarProps;
 export const CommandBarButtonsMenu = NamedFC<CommandBarButtonsMenuProps>(
     'CommandBarButtonsMenu',
     props => {
-        const onRenderItem = (item: IOverflowSetItemProps): JSX.Element => {
```

Comment:
Are we getting rid of this because we don't really need it/can utilize the default behaviors of the control?

Reply:
We don't need it, the component was already using the onRender functions in the overflowItem objects themselves. I think onRenderItem is intended to be used for items in an OverflowSet that aren't in the overflow dropdown.